hack/build-antrea-linux-all.sh

```diff
@@ -112,11 +112,11 @@ GO_VERSION=$(head -n 1 build/images/deps/go-version)
 # new base images in the build chain.
 if $PULL; then
     if [[ ${DOCKER_REGISTRY} == "" ]]; then
-        docker pull $PLATFORM_ARG ubuntu:20.04
+        docker pull $PLATFORM_ARG ubuntu:22.04
```

Comment:
I searched `20.04`, there are actually a few more files using `ubuntu:20.04`, e.g.:`Dockerfile.octant.ubuntu`, `flow-aggregator/Dockerfile`. Should we upgrade them as well to keep base image consistent? or you feel it's OK to keep using `ubuntu:20.04` if they are not impacted by old iptables?

Reply:
that's intentional, I'm planning to update the rest of the images in a follow-up PR